code/drasil-docLang/lib/Drasil/DocumentLanguage/Definitions.hs

```diff
@@ -162,42 +164,41 @@ buildDDescription' Verbose  u d m = [UlC . ulcc . Enumeration $ Definitions $
 
 -- | Create the fields for a general definition from a 'GenDefn' chunk.
 mkGDField :: GenDefn -> SystemInformation -> Field -> ModRow -> ModRow
-mkGDField g _ l@Label fs = (show l, [mkParagraph $ atStart g]) : fs
+mkGDField g _ l@Label fs = (getSent l, [mkParagraph $ atStart g]) : fs
 mkGDField g _ l@Units fs =
-  maybe fs (\udef -> (show l, [mkParagraph . Sy $ usymb udef]) : fs) (getUnit g)
-mkGDField g _ l@DefiningEquation fs = (show l, [unlbldExpr $ express g]) : fs
-mkGDField g m l@(Description v u) fs = (show l,
+  maybe fs (\udef -> (getSent l, [mkParagraph . Sy $ usymb udef]) : fs) (getUnit g)
+mkGDField g _ l@DefiningEquation fs = (getSent l, [unlbldExpr $ express g]) : fs
+mkGDField g m l@(Description v u) fs = (getSent l,
   buildDescription v u (express g) m []) : fs
-mkGDField g m l@RefBy fs = (show l, [mkParagraph $ helperRefs g m]) : fs --FIXME: fill this in
-mkGDField g _ l@Source fs = (show l, helperSources $ g ^. getDecRefs) : fs
-mkGDField g _ l@Notes fs = nonEmpty fs (\ss -> (show l, map mkParagraph ss) : fs) (g ^. getNotes)
-mkGDField _ _ l _ = error $ "Label " ++ show l ++ " not supported for gen defs"
+mkGDField g m l@RefBy fs = (getSent l, [mkParagraph $ helperRefs g m]) : fs --FIXME: fill this in
+mkGDField g _ l@Source fs = (getSent l, helperSources $ g ^. getDecRefs) : fs
+mkGDField g _ l@Notes fs = nonEmpty fs (\ss -> (getSent l, map mkParagraph ss) : fs) (g ^. getNotes)
+mkGDField _ _ l fs = (getSent l +:+ S ":Label not supported for data definitions", []) : fs
```

Comment:
Instead of "gen defs", like was used before, I would put
```suggestion
mkGDField _ _ l fs = (getSent l +:+ S ":Label not supported for general definitions", []) : fs
```

Reply:
I think for these I copy pasted the format and just forgot to adjust the error message, will push the fix in a minute